### ISSUE TITLE
Persist configuration options to disk

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo_gui#375]
#372 reminded me of an issue we should track (and probably needs some discussion). We have some configuration options now, accessible through the gear in the top toolbar. Right now, it's all client-side, so it works for the current session and that's it. It should work forever! To do this, we need to persist something to disk, which means that we need to let Python know about the configuration stuff and save it somewhere.

A few discussion points.

1. Where should the configuration be saved?
2. What format should we use?

I think the `.ini` style configuration that we use in Nengo is probably a decent enough format to use. We could also use where Nengo saves configuration:

```ipython
In [2]: nengo.utils.paths.config_dir
Out[2]: '/Users/Trevor/.config/nengo'
```

I'm okay with using it directly, but it could be the `nengo_gui` equivalent too.

**Comment by jgosmann** [comment]
I would vote for separate configuration files. Seems cleaner to me.

**Comment by tbekolay** [comment]
Oh yes! I agree, separate configuration files (not using `nengorc` for sure), just using the same format and storing the file (`nengo_gui.conf` or something?) in the same directory (`~/.config/nengo`).

**Comment by mundya** [comment]
FWIW `nengo_spinnaker` config sneaks itself into `~/.config/nengo` but will follow whatever convention is set here...

**Comment by jgosmann** [comment]
> just using the same format and storing the file (nengo_gui.conf or something?) in the same directory (~/.config/nengo)

I'm ok with that.

**Comment by rfernand2** [comment]
Persisting the font size will make it easier for Nengo instructors to keep the font large enough for their audiences.  :)

**Comment by Seanny123** [comment]
+1 for same directory as `nengo`

**Comment by tcstewar** [comment]
This was solved in #649 by persisting to the javascript localstorage!  Much nicer than figuring out where to store it ourselves...
